python/meterpreter/ext_server_stdapi.py

```diff
@@ -1326,6 +1328,13 @@ def stdapi_fs_file_copy(request, response):
     shutil.copyfile(unicode(oldname), unicode(newname))
     return ERROR_SUCCESS, response
 
+@register_function_if(sys.platform == 'darwin' or sys.platform.startswith('linux'))
```

Comment:
shouldn't `os.chmod` work also on windows machines?

Reply:
I'm not sure of the behavior on windows: https://superuser.com/questions/106181/equivalent-of-chmod-to-change-file-permissions-in-windows